modules/data/data_reader.py

```diff
@@ -76,9 +76,54 @@ def load_configuration(path: str, load_default: bool = True, force_load: bool =
     return conf
 
 
+def read_env(config: dict):
+    """Reads the enviroment variables and stores the values in the config dict.
+    Any key already present will be overwritten
+
+    Args:
+        config: configuration dictionary
+    """
+    new_vars = {}
+    config['test'] = config['test'] if config.get('test', False) else {}
+    config['meme'] = config['meme'] if config.get('meme', False) else {}
```

Comment:
:thinking: why an empty dict {}? should be `False`?

Reply:
The default False value is used to trigger the ternary operator, but then i need an empty dictionary, since in the following lines i use the **['meme']** and the **['test']** sub-dictionaries to add the settings.